include/Enums.h

```diff
@@ -255,15 +255,19 @@ static const std::string TurbulenceModelConstantNames[] = {
 enum ActuatorType {
   ActLinePointDrag = 0,
   ActLineFAST = 1,
-  ActDiskFAST = 2,
+  ActLineFASTNGP = 2,
+  ActDiskFAST = 3,
+  ActDiskFASTNGP = 4,
   ActuatorType_END
 };
 
- static std::map<std::string, ActuatorType> ActuatorTypeMap = {
-     {"ActLinePointDrag",ActuatorType::ActLinePointDrag},
-     {"ActLineFAST",ActuatorType::ActLineFAST},
-     {"ActDiskFAST",ActuatorType::ActDiskFAST}
- };
+static std::map<std::string, ActuatorType> ActuatorTypeMap = {
+  {"ActLinePointDrag",ActuatorType::ActLinePointDrag},
```

Comment:
I kept it as a default for ‘ActuatorBulk‘ and ‘ActuatorMeta’ since they are the base classes and don’t rely on OpenFAST. Also we could use it for the fixed wing stuff Lawrence is working on. But I haven’t looked at his code yet so I’m not sure if he used it or created his own type. I’m not opposed to getting rid of it, but it does serve a purpose currently.